backend/db/refresh.py

```diff
@@ -33,6 +40,26 @@ def trigger_resolve_failures(
     if resolve_fetch_failures_0_members:
         resolve_failures_0_members_if_exist(local)
 
+def _calc_refresh_datetime(
+    con: Connection, since: str, buffer_hours=DEFAULT_BUFFER_HOURS, force_non_contiguity=False
+) -> str:
+    """Calculate the datetime from which to fetch data since."""
```

Comment:
## Add: `YYYY-MM-DD` support
Previously had to be formatted as: `YYYY-MM-DDTHH:MM:SS.SSSSSS+HH:MM`